src/Core/Style.js

```diff
@@ -446,6 +448,8 @@ class Style {
      * @returns {Style}
      */
     setFromGeojsonProperties(properties, type) {
+        // TODO !! FT GC
```

Comment:
What does FT GC means? Also, I don't understand what's to be done, could you explain a bit more please?